query/aql_compiler.go

```diff
@@ -678,6 +687,11 @@ func (qc *AQLQueryContext) Rewrite(expression expr.Expr) expr.Expr {
 			return expression
 		}
 
+		if err := blockInt64(e.LHS, e.RHS); err != nil {
```

Comment:
maybe a TODO here to mark that we currently block binary operation on int64 types